packages/developer-portal/src/components/pages/app-detail/pipeline/create-pipeline.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState } from 'react'
+import { cx } from '@linaria/core'
+import {
+  BodyText,
+  Button,
+  ButtonGroup,
+  ColSplit,
+  elM6,
+  elMb10,
+  elMt10,
+  FlexContainer,
+  FormLayout,
+  Grid,
+  Input,
+  InputAddOn,
+  InputGroup,
+  InputWrap,
+  InputWrapFull,
+  Label,
+  Modal,
+  PersistantNotification,
+  Subtitle,
+  Title,
+} from '@reapit/elements'
+import { IconContainer } from '../../webhooks/__styles__'
+import { WebhooksAnimatedNewIcon } from '../../webhooks/webhooks-animated-new-icon'
+import { WebhooksAnimatedDocsIcon } from '../../webhooks/webhooks-animated-docs-icon'
+import { ExternalPages, openNewPage } from '@/utils/navigation'
+import { mixed, object, string } from 'yup'
+import { AppTypeEnum, PackageManagerEnum, PipelineModelInterface } from '@reapit/foundations-ts-definitions'
+import { httpsUrlRegex, UpdateActionNames } from '@reapit/utils-common'
+import { useForm } from 'react-hook-form'
+import { yupResolver } from '@hookform/resolvers/yup'
+import errorMessages from '@/constants/error-messages'
+import { useReapitUpdate } from '@reapit/utils-react'
+import { reapitConnectBrowserSession } from '../../../../core/connect-session'
```

Comment:
Should come from @reapit/connect-session

Reply:
that's `ReapitConnectBrowserSession` which is the class that's being created from core/connect-session